dashboard/app/views/layouts/_header.html.haml

```diff
@@ -25,6 +25,18 @@
               = render partial: 'shared/header_progress'
             - elsif @edit_blocks
               .header_level{style:'font-size: 250%; line-height: 125%;'} Editing #{@edit_blocks.humanize}
-          - if params[:controller] !~ /^sessions/
-            .header_user
-              = render partial: 'shared/user_header'
+          .header_right
+            %span.stage_links{style: 'display: none;'}
```

Comment:
not sure if stage_links sounds like the right name for this... standalone_level_links?

IIRC stage_links was used to share CSS with something else but it doesn't seem to be used anymore.